pkg/cmd/server/origin/master_config.go

```diff
@@ -568,9 +568,9 @@ func (c *MasterConfig) DeploymentControllerClients() (*osclient.Client, *kclient
 	return osClient, kClient
 }
 
-// DeployerPodControllerClients returns the deployer pod controller client objects
-func (c *MasterConfig) DeployerPodControllerClients() (*osclient.Client, *kclient.Client) {
-	return c.PrivilegedLoopbackOpenShiftClient, c.PrivilegedLoopbackKubernetesClient
+// DeployerPodControllerClients returns the deployer pod controller client object
+func (c *MasterConfig) DeployerPodControllerClient() *kclient.Client {
```

Comment:
Since you're touching it, how about either converting to an SA or opening an issue where you promise to convert it?